source/code/source/Entity.h.ejs

```diff
@@ -46,8 +47,9 @@ public:
     // EntityToken if the call succeeds.
     AsyncOp<void> GetEntityToken(const PlayFabAuthenticationGetEntityTokenRequest& request, const TaskQueue& queue);
 
-    // Non-generated API
+    // Non-generated APIs
     EventManager::EventManagerAPI eventManagerAPI;
+    QoS::QoSAPI const QoSAPI;

```

Comment:
nit: m_  on these

Reply:
Left off intentionally since these are public members.  Could easily change to private member and have a public accessor but since the APIs are const anyway this seemed fine.